web-app/src/Home.js

```diff
@@ -183,10 +192,11 @@ useEffect(() => {
         owner,
         title,
         description,
-        startDate: new Date(startDate).toISOString(),
```

Comment:
We need this conversion to send the date in the format that the back-end is expecting